src/periphery/interfaces/ISablierMerkleLL.sol

```diff
@@ -32,5 +32,6 @@ interface ISablierMerkleLL is ISablierMerkleBase {
 
     /// @notice The start time, cliff duration and the end duration used to calculate the time variables in
     /// `LockupLinear.CreateWithTimestamps`.
+    /// @dev A start time value of zero will be considered as `block.timestamp`.
```

Comment:
i didn't add this here, as it was included in the `MerkleLL.Schedule` struct natspec, but we can have it in both places